tensorrt_llm/_torch/models/modeling_llama.py

```diff
@@ -324,14 +812,20 @@ def __init__(
         self.layer_idx = layer_idx
         self.is_quanted = model_config.quant_config and model_config.quant_config.quant_mode.has_any_quant(
         )
+        self.is_fp8_quant = self.is_quanted and model_config.quant_config.quant_mode.has_fp8_qdq(
```

Comment:
Is it possible to leave the original `Llama4DecoderLayer` and the other modules your modified alone and add min-latency versions of these in another file named `modeling_llama_min_latency.py`? You can add a `PyTorchConfig.enable_min_latency` config. When it's on, you can use `MinLatencyLlama4DecoderLayer` on line 1277 in this file, and when it's off, use `Llama4DecoderLayer`. The goal is to contain the min-latency code so it doesn't break the other scenarios of llama4, especially max-throughput. You'll need to add e2e tests to cover the min-latency code path. What do you think @nvpohanh ? cc @QiJune

Reply:
I am fine with this idea, but I need @QiJune to confirm that it is okay to add the `enable_min_latency` field into `PyTorchConfig`. 

cc @chenfeiz0326

Reply:
Yes, I am okay with `enable_min_latency` field.